packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -479,47 +481,43 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 		}
 
 		const resolve = localOpMetadata as PendingAliasResolve;
-		const aliasResult = this.processAliasMessageCore(aliasMessage);
+		const aliasResult = this.processAliasMessageCore(
+			aliasMessage.internalId,
+			aliasMessage.alias,
+		);
 		if (local) {
 			resolve(aliasResult);
 		}
 	}
 
-	public processAliasMessageCore(aliasMessage: IDataStoreAliasMessage): boolean {
-		if (this.alreadyProcessed(aliasMessage.alias)) {
+	public processAliasMessageCore(internalId: string, alias: string): boolean {
```

Comment:
If the intention here is to not expose `IDataStoreAliasMessage`, I think for all intents and purposes we'll already leak its logic so we should just do it explicitly.

Reply:
Yes, I was trying to reduce number of types that we have to export / expose. Maybe we will have to do it anyway in the future - not sure, but for now, the less types exported - the better!